setup.py

```diff
@@ -26,8 +26,18 @@
     # - stdeb.cfg
     'version': '3.0.1-master',
     'packages': find_packages(exclude=['test']),
+    'package_data': {
+        'ros_buildfarm.templates': [
+            '*/*/*.css',
+            '*.em',
```

Comment:
not critical but any reason not use `**/*.em` to select all .em files no matter the directory structure?

Mostly applicable to .em since it have 3 entries but for the other file extensions if we don't need to select some of them in and some of them out, could be less error prone not to take care of detailing the number of subdirs in the epxression.

Reply:
This is exactly what broke the Python 3.6 builds in CI: https://github.com/ros-infrastructure/ros_buildfarm/pull/1039#issuecomment-2123703831